### PR TITLE
Update outdated files in the dev-1.19-ko.7 branch

### LINKED ISSUE #25522: Outdated files in the dev-1.19-ko.7 branch
# This is a Bug Report
## Problem
Outdated files in the dev-1.19-ko.7 branch.

### 19 files to be modified 
1. [ ] content/en/docs/concepts/cluster-administration/logging.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/cluster-administration/networking.md 1 file changed, 5 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/concepts/cluster-administration/system-metrics.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/configuration/secret.md 1 file changed, 7 insertions(+), 8 deletions(-)
1. [ ] content/en/docs/concepts/containers/images.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 39 insertions(+), 19 deletions(-)
1. [ ] content/en/docs/concepts/overview/working-with-objects/labels.md 1 file changed, 5 insertions(+), 4 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/ingress-controllers.md 1 file changed, 28 insertions(+), 28 deletions(-)
1. [ ] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 31 insertions(+), 22 deletions(-)
1. [ ] content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 6 deletions(-)
1. [ ] content/en/docs/reference/command-line-tools-reference/feature-gates.md 1 file changed, 4 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/setup/best-practices/multiple-zones.md 1 file changed, 3 insertions(+), 3 deletions(-)
1. [ ] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/examples/application/job/cronjob.yaml 1 file changed, 1 insertion(+)
1. [ ] content/en/examples/application/mysql/mysql-configmap.yaml 1 file changed, 4 insertions(+), 4 deletions(-)
1. [ ] content/en/examples/application/mysql/mysql-services.yaml 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/examples/application/mysql/mysql-statefulset.yaml 1 file changed, 5 insertions(+), 5 deletions(-)

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `dev-1.19-ko.7` branch.

For example:
```
# checkout `dev-1.19-ko.7`
...
# check what is updated in the upstream 
git diff dev-1.19-ko.6 dev-1.19-ko.7 -- content/en/docs/concepts/cluster-administration/logging.md
# apply changes to content/ko
vi content/ko/docs/concepts/cluster-administration/logging.md
...
# commit and push
...
# make PR to `dev-1.19-ko.7`
```

## Pages to Update
Korean pages.

### PR DESCRIPTION
**Pull request by pjhwa**
Closes #25522 

Update outdated files in the upstream/dev-1.19-ko.7 branch.

16 files are updated
- [x] content/en/docs/concepts/cluster-administration/networking.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [x] content/en/docs/concepts/cluster-administration/system-metrics.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/configuration/secret.md 1 file changed, 7 insertions(+), 8 deletions(-)
- [x] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 39 insertions(+), 19 deletions(-)
- [x] content/en/docs/concepts/overview/working-with-objects/labels.md 1 file changed, 5 insertions(+), 4 deletions(-)
- [x] content/en/docs/concepts/services-networking/ingress-controllers.md 1 file changed, 28 insertions(+), 28 deletions(-)
- [x] content/en/docs/concepts/storage/persistent-volumes.md 1 file changed, 31 insertions(+), 22 deletions(-)
- [x] content/en/docs/concepts/workloads/controllers/daemonset.md 1 file changed, 6 deletions(-)
- [x] content/en/docs/reference/command-line-tools-reference/feature-gates.md 1 file changed, 4 insertions(+), 2 deletions(-)
- [x] content/en/docs/reference/kubectl/cheatsheet.md 1 file changed, 1 insertion(+)
- [x] content/en/docs/setup/production-environment/container-runtimes.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/setup/production-environment/windows/user-guide-windows-containers.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/examples/application/job/cronjob.yaml 1 file changed, 1 insertion(+)
- [x] content/en/examples/application/mysql/mysql-configmap.yaml 1 file changed, 4 insertions(+), 4 deletions(-)
- [x] content/en/examples/application/mysql/mysql-services.yaml 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/examples/application/mysql/mysql-statefulset.yaml 1 file changed, 5 insertions(+), 5 deletions(-)

3 files are already updated in Korean documents
- [x] content/en/docs/concepts/cluster-administration/logging.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/containers/images.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/setup/best-practices/multiple-zones.md 1 file changed, 3 insertions(+), 3 deletions(-)

### CODE REVIEW

File: content/ko/examples/application/mysql/mysql-statefulset.yaml [hunk 1]
```diff
@@ -105,22 +105,22 @@ spec:
           set -ex
           cd /var/lib/mysql
 
-          # Determine binlog position of cloned data, if any.
+	      # 복제된 데이터의 binlog 위치를 확인.
```
Comment (pjhwa) [inline content/ko/examples/application/mysql/mysql-statefulset.yaml:108]:
```suggestion
          # 복제된 데이터의 binlog 위치를 확인.
```

File: content/ko/examples/application/mysql/mysql-statefulset.yaml [hunk 2]
```diff
@@ -105,22 +105,22 @@ spec:
...
           if [[ -f xtrabackup_slave_info && "x$(<xtrabackup_slave_info)" != "x" ]]; then
-            # XtraBackup already generated a partial "CHANGE MASTER TO" query
-            # because we're cloning from an existing slave. (Need to remove the tailing semicolon!)
+		    # XtraBackup은 기존 레플리카에서 복제하기 때문에
+		    # 일부 "CHANGE MASTER TO" 쿼리는 이미 생성했음. (테일링 세미콜론을 제거해야 한다!)
```
Comment (pjhwa) [inline content/ko/examples/application/mysql/mysql-statefulset.yaml:111]:
```suggestion
            # XtraBackup은 기존 레플리카에서 복제하기 때문에
            # 일부 "CHANGE MASTER TO" 쿼리는 이미 생성했음. (테일링 세미콜론을 제거해야 한다!)
```

File: content/ko/examples/application/mysql/mysql-statefulset.yaml [hunk 3]
```diff
@@ -105,22 +105,22 @@ spec:
...
             cat xtrabackup_slave_info | sed -E 's/;$//g' > change_master_to.sql.in
-            # Ignore xtrabackup_binlog_info in this case (it's useless).
+            # 이 경우에는 xtrabackup_binlog_info는 무시(필요없음).
             rm -f xtrabackup_slave_info xtrabackup_binlog_info
           elif [[ -f xtrabackup_binlog_info ]]; then
-            # We're cloning directly from master. Parse binlog position.
+		    # Primary로부터 직접 복제함. binlog 위치를 파싱.
```
Comment (pjhwa) [inline content/ko/examples/application/mysql/mysql-statefulset.yaml:116]:
```suggestion
            # Primary로부터 직접 복제함. binlog 위치를 파싱.
```

File: content/ko/examples/application/mysql/mysql-statefulset.yaml [hunk 4]
```diff
@@ -133,11 +133,11 @@ spec:
                           MASTER_PASSWORD='', \
                           MASTER_CONNECT_RETRY=10; \
                         START SLAVE;" || exit 1
-            # In case of container restart, attempt this at-most-once.
+			# 컨테이너가 다시 시작하는 경우, 이 작업을 한번만 시도한다.
```
Comment (pjhwa) [inline content/ko/examples/application/mysql/mysql-statefulset.yaml:136]:
```suggestion
            # 컨테이너가 다시 시작하는 경우, 이 작업을 한번만 시도한다.
```

File: content/ko/docs/reference/command-line-tools-reference/feature-gates.md [hunk 5]
```diff
@@ -380,9 +380,10 @@ kubelet과 같은 컴포넌트의 기능 게이트를 설정하려면, 기능 
    자세한 내용은 [원시 블록 볼륨 지원](/ko/docs/concepts/storage/persistent-volumes/#원시-블록-볼륨-지원)을
    참고한다.
 - `BoundServiceAccountTokenVolume`: ServiceAccountTokenVolumeProjection으로 구성된 프로젝션 볼륨을 사용하도록 서비스어카운트 볼륨을
-   마이그레이션한다.
-   자세한 내용은 [서비스 어카운트 토큰 볼륨](https://git.k8s.io/community/contributors/design-proposals/storage/svcacct-token-volume-source.md)을
-   확인한다.
+   마이그레이션한다. 클러스터 관리자는 `serviceaccount_stale_tokens_total` 메트릭을
+   사용하여 확장 토큰에 의존하는 워크로드를 모니터링할 수 있다. 이러한 워크로드가
+   없는 경우 `--service-account-extend-token-expiration=false` 플래그로 `kube-apiserver`를 시작하여 확장 토큰을 끈다.
+   [바인딩된 서비스 어카운트 토큰]을 확인한다.
```
Comment (seokho-son) [inline content/ko/docs/reference/command-line-tools-reference/feature-gates.md:386]:
```suggestion
   [바인딩된 서비스 어카운트 토큰](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/1205-bound-service-account-tokens/README.md)을 확인한다.
```

링크가 누락되어 있습니다 ^^

**Review by seokho-son — COMMENTED** [review]
@pjhwa Thank you for maintaining Korean dev branch !!
This PR is the final PR for release 1.19 !

I have a minor comment on this PR.
Please apply it and squash the commits.

File: content/ko/docs/reference/command-line-tools-reference/feature-gates.md (hunk 5 above)
Comment (pjhwa) [reply content/ko/docs/reference/command-line-tools-reference/feature-gates.md:386]: 반영하겠습니다. ^^

**Comment by pjhwa** [comment]
> @pjhwa Thank you for maintaining Korean dev branch !!
> This PR is the final PR for release 1.19 !
> 
> I have a minor comment on this PR.
> Please apply it and squash the commits.

Thank you for your review!

**Comment by seokho-son** [comment]
Thank you !
/lgtm
/approve

**Pull request merged**